llvm/lib/Transforms/Vectorize/SandboxVectorizer/SeedCollector.cpp

```diff
@@ -22,6 +22,23 @@ namespace llvm::sandboxir {
 cl::opt<unsigned> SeedBundleSizeLimit(
     "sbvec-seed-bundle-size-limit", cl::init(32), cl::Hidden,
     cl::desc("Limit the size of the seed bundle to cap compilation time."));
+cl::opt<bool>
+    DisableStoreSeeds("sbvec-disable-store-seeds", cl::init(false), cl::Hidden,
+                      cl::desc("Don't collect store seed instructions."));
+cl::opt<bool>
+    DisableLoadSeeds("sbvec-disable-load-seeds", cl::init(true), cl::Hidden,
+                     cl::desc("Don't collect load seed instructions."));
+
+#define LoadSeedsDef "loads"
+#define StoreSeedsDef "stores"
+cl::opt<std::string>
+    ForceSeed("sbvec-force-seeds", cl::init(""), cl::Hidden,
+              cl::desc("Enable only this type of seeds. This can be one "
+                       "of: '" LoadSeedsDef "','" StoreSeedsDef "'."));
```

Comment:
I am trying to think of a better way of specifying the seeds to be collected with just one flag instead of having a couple of disable flags and a force flag. Perhaps replace them with a single cl::opt that uses the textual form of the ForceSeed flag, but also specifies the default values? Wdyt ?

Reply:
Done. This new method is a little loose with the parsing. Someone could use --sbvec-collect-seeds=asdfasdasfdas and wouldn't get a syntax error, but seems likely ok for a developer option?